easybuild/easyconfigs/s/Siesta/Siesta-4.0-b2_fixes.patch

```diff
@@ -0,0 +1,11 @@
+--- Src/Makefile.orig	2016-05-18 10:29:01.484660260 +0800
```

Comment:
@migueldiascosta please mention what this patch does + patch author above this line?